common/interpreter/wawaka_wasm/README.md

```diff
@@ -16,81 +16,69 @@ set of supporting functions that can be executed inside the contract
 enclave. As such contracts can be implemented in any programming
 language that can be compiled into WASM.
 
-## Building Wawaka ##
+## Build Wawaka ##
 
 The Wawaka interpreter is not built by default. To build a contract
 enclave with Wawaka enabled, you will need to do the following:
 
-  * Install and configure [emscripten](https://emscripten.org/)
+  * Install and configure the WASM development toolchain
   * Pull the WAMR submodule (if the repo was not cloned with the `--recurse-submodules` flag)
   * Set the `PDO_INTERPRETER` environment variable to `wawaka`
 
-### Install emscripten ###
+### Install WASM Development Toolchain ###
 
-There are many toolchains that could be used to build a WASM code. We have tested (and our sample
-and test contracts use) [emscripten](https://emscripten.org/). Note that we currently require the `fastcomp` compiler which is no longer the default compiler.
+There are many toolchains that could be used to build a WASM code. By default, Wawaka contracts are
+compiled with the compilers provided by [WASI SDK](https://github.com/WebAssembly/wasi-sdk). To use
+WASI SDK, download and install the appropriate package file from
+https://github.com/WebAssembly/wasi-sdk/releases (we have verified that release wasi-sdk-12 works
+with WAMR version WAMR-03-25-2021).
 
 ```bash
-cd ${PDO_SOURCE_ROOT}
-
-git clone https://github.com/emscripten-core/emsdk.git
-cd ${PDO_SOURCE_ROOT}/emsdk
-git checkout 1.39.20 # last release to support fastcomp backend
-
-./emsdk install latest-fastcomp
-./emsdk activate latest-fastcomp
-
-source ./emsdk_env.sh
+wget -q -P /tmp https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-12/wasi-sdk_12.0_amd64.deb
+sudo dpkg --install /tmp/wasi-sdk_12.0_amd64.deb
 ```
 
-### WAMR setup ###
+These directions assume that the SDK will be installed in the PDO source tree. Typically, the WASI
+SDK would be installed in the directory `/opt/wasi-sdk`. Please set the environment variable
+`WASI_SDK_DIR` to the directory where the SDK is installed.
```

Comment:
Although this works, the `WASI_SDK_DIR` environment variable set by the user is ignored.
In fact, the default value of the `WASI_SDK_DIR` _cmake_ variable (i.e., `/opt/wasi-sdk`) is always set by the WAMR toolchain.

So, again,
* either we set the cmake variable in the Makefile (and the WAMR toolchain will use it),
* or (suggestion below) we remove the environment variable as it is not required. So we will be relying on the fact that the debian package installation folder and the default value in the WAMR toolchain match.

```suggestion
These directions assume that the WASI SDK will be installed in the default directory `/opt/wasi-sdk`.
```

Reply:
you should never set wasi_sdk_dir. it is not used at all.